app/models/event.rb

```diff
@@ -12,6 +12,8 @@
 class Event < ActiveRecord::Base
   translates :title, :introduction, :conclusion
   belongs_to :location
+  # suggestion from Nicholas
+  to_param :title
```

Comment:
@nicholasjhenry If we are going to use `to_param` to generate pretty urls for pages representing objects in the application, it might be useful to define a common standard (mixin?)

Also, in the case of that particular implementation, I think that it would be something more like:
```ruby
def to_param
  "#{id} #{title}".parameterize
end
```
Allowing us to have urls like these: `/events/1-my-event-title` where Rails would automatically be able to load the `Event` object like so `event = Event.find("1-my-event-title")`

Finally, because I know you LOVE gems, would there be any added value for us to use a specialized pretty urls library like https://github.com/norman/friendly_id ?